enterprise/views.py

```diff
@@ -572,6 +583,24 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_customer, co
         if course_details['start']:
             course_start_date = parse(course_details['start']).strftime('%B %d, %Y')
 
+        duration = _('12 weeks per course')
```

Comment:
Let's remove duration for now. I will create a followup story for implementing it dynamically.